File: src/dto/SimplePostDto.js
export class SimplePostDto {
  constructor(content) {
    this.id = content.id;
    this.no = content.no;
    this.title = content.title;
    this.commentSize = content.commentSize;
    this.writer = content.writer;
    this.hit = content.hit;
    this.time = content.time;
    this.hasImage = content.hasImage;
  }
}

File: backend/src/main/java/com/yellowsunn/spring_security/domain/dto/SimplePostDto.java
package com.yellowsunn.spring_security.domain.dto;

import lombok.Builder;
import lombok.Data;

// 간단한 게시글 정보를 담은 DTO
@Data
@Builder
public class SimplePostDto {

    private Long id;
    private Long no;
    private String title;
    private int commentSize;
    private String writer;
    private Long hit;
    private String time;
    private Boolean hasImage;
}
